rllib/core/learner/learner.py

```diff
@@ -893,7 +896,7 @@ def compute_loss_for_module(
 
         Think of this as computing loss for a single agent. For multi-agent use-cases
         that require more complicated computation for loss, consider overriding the
-        `compute_loss` method instead.
+        `compute_losses` method instead.
```

Comment:
Can `compute_losses` call internally `compute_loss_for_module` or is the latter called nevertheless and losses would be computed two times?

Reply:
The default implementation of `compute_losses` calls n times `compute_loss_for_module`, where n is the number of RLModules within the Learner's MultiRLModule. So nothing is computed twice.

If you have a complex multi-agent case, you should override `compute_losses`, in which case the n calls to `compute_loss_for_module` will NOT be made.